src/bot.py

```diff
@@ -24,17 +26,24 @@
 
 def main():
     """Start the bot."""
+
     # Set use_context=True to use the new context based callbacks
     updater = Updater(token=os.getenv('RC_DINING_BOT_TOKEN'), use_context=True)
-
-    # Get the dispatcher to
     dispatcher = updater.dispatcher
 
     # on different commands - answer in Telegram
     dispatcher.add_handler(CommandHandler(START, handle_start))
     dispatcher.add_handler(CommandHandler(HELP, handle_help))
     dispatcher.add_handler(CommandHandler(BREAKFAST, handle_menu(meal=BREAKFAST)))
     dispatcher.add_handler(CommandHandler(DINNER, handle_menu(meal=DINNER)))
+    dispatcher.add_handler(CommandHandler(SETTINGS, handle_settings))
+    dispatcher.add_handler(CommandHandler(HIDE_CUISINE, handle_hidden_cuisine))
+
+    # add callback_query handler
+    dispatcher.add_handler(CallbackQueryHandler(handle_start, pattern='start.+'))
```

Comment:
should we add '^' to the regex to ensure it starts with the pattern?

Reply:
sure.